File: src/main/java/com/example/kehtolaulu/nanoblogs/test/stories/NewStoryPage.java
package com.example.kehtolaulu.nanoblogs.test.stories;

import com.example.kehtolaulu.nanoblogs.test.Page;
import org.openqa.selenium.By;

public class NewStoryPage extends Page {
    @Override
    public void visit() {
        navigation().visitNewStoryPage();
    }

    public void createStory(StoryData storyData) {
        input(storyData.getText()).toElement(By.id("story_content"));
        clickElement(By.name("commit"));
    }
}

File: src/main/java/com/example/kehtolaulu/nanoblogs/test/stories/NewStoryPage.kt
package com.example.kehtolaulu.nanoblogs.test.stories

import com.example.kehtolaulu.nanoblogs.test.Page
import org.openqa.selenium.By

class NewStoryPage : Page() {
    override fun visit() {
        navigation().visitNewStoryPage()
    }

    fun createStory(storyData: StoryData) {
        input(storyData.text).toElement(By.id("story_content"))
        clickElement(By.name("commit"))
    }
}